article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -55,29 +55,30 @@ Bibliography:
 
 # Introduction
 
-Diffusion-weighted Magnetic Resonance Imaging (DW-MRI) is a non-invasive
-biomedical imaging technique that allows us to infer properties of brain tissue
-microstructure in vivo. Diffusion tensor imaging (DTI), one of the most
-commonly used DW-MRI techniques, models diffusion anisotropy of tissues using a
+Diffusion-weighted Magnetic Resonance Imaging (DW-MRI)  is a
+biomedical imaging technique that allows the non-invasive acquisition of in vivo data
+from which tissue microstructure can be inferred. Diffusion tensor imaging (DTI), one of the most
+commonly used DW-MRI techniques in the brain, models diffusion anisotropy of tissues using a
 second-order tensor known as the diffusion tensor (DT) [@Basser1994-zd], [@Basser1994-hg].
 DTI-based measures such as fractional anisotropy (FA) and mean diffusivity (MD)
-are normally used to assess properties of brain microstructure. For example, FA is thought to be an indicator of different microstructural properties:
+are normally used to assess properties of brain microstructure.
+For example, FA is thought to be an indicator of different microstructural properties:
 packing density of axons, and the density of myelin in nerve fibers [@Beaulieu2002-tl],
 but also indicates white matter coherence -- the alignment of axons within a measurement voxel.
-However, because a measurement voxel can contain partial volumes of different
-types of tissue, these measures are not always specific to one particular type of tissue.
-For example, diffusion anisotropy in regions near cerebral ventricle and parenchyma can be
+However, because a measurement voxel can contain multiple
+types of tissue, these measures are not always specific to one particular type of tissue, an effect called partial voluming.
```

Comment:
IMO I think it's good to say partial volume effect because it emphasizes that it's a known and common problem. But I'm fine with the original version too.

Overall it looks good to me, nice work!